README.md

```diff
@@ -422,7 +422,7 @@ A `PipelineRun` is how you can start a pipeline and tie it to the git and image
 $ tkn pipeline start build-and-deploy \
     -r git-repo=api-repo \
     -r image=api-image \
-    -p deployment-name=vote-api
```

Comment:
There are PRs in progress renaming the deployments - https://github.com/openshift-pipelines/vote-api/pull/2 and https://github.com/openshift-pipelines/vote-ui/pull/4

@sthaha @praveen4g0 why haven't they been merged yet?

Reply:
The renaming was the only intended change. The rest is just a mistake. Forgot to switch branch for my personal tests.